discovery-infra/test_infra/helper_classes/infra_env.py

```diff
@@ -8,28 +7,23 @@
 from test_infra import consts, utils
 from test_infra.assisted_service_api import InventoryClient, models
 from test_infra.helper_classes.config import BaseInfraEnvConfig
+from test_infra.helper_classes.entity import Entity
 from test_infra.helper_classes.nodes import Nodes
 
 
-class InfraEnv:
+class InfraEnv(Entity):
```

Comment:
can we have some kind of field to have this in a simpler form:
https://github.com/openshift/assisted-test-infra/blob/master/discovery-infra/test_infra/controllers/node_controllers/terraform_controller.py#L55

for example, if we had ``template = "infra-env"`` in this class and ``template = "baremetal"`` in the base class this could have been simpler than using ``isinstance``

Reply:
Not sure I fully understand what you mean. Add another field to the configuration? Or to the class?

Reply:
to the configuration class